packages/scene/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@behave-graph/scene",
+  "version": "0.9.12",
```

Comment:
curious if it makes sense to version this at 0.0.1 so we have more room to move up?

Reply:
It doesn't really affect as we can still use `0.10.x`, `0.11.x` and so on. Personally I would try using the same version for all the packages in the same repository, that would help to make sure which version should be used with the `core`. ie. `@behave-graph/scene@0.9.12` is fully compatible  with `@behave-graph/core@0.9.12`. As an example you can see the [angular repository](https://github.com/angular/angular). Right now is not something we should worry about, and it doesn't look ergonomic to upgrade the scene package if there are no changes on it, but it will make really sense in a future. I personally have worked in both cases of versioning in monorepos and the different versioning for each package ends up being a pain in the ass when you have breaking changes

Reply:
keeping same version makes sense @aitorllj93